src/createContext.ts

```diff
@@ -473,7 +637,7 @@ const createContext = <T>(
   externalContext?: T,
   externalBuiltIns: CustomBuiltIns = defaultBuiltIns
 ): Context => {
-  if (chapter === Chapter.FULL_JS || chapter === Chapter.FULL_TS) {
+  if (chapter === Chapter.FULL_JS || chapter === Chapter.FULL_TS || chapter === Chapter.PYTHON_1) {
```

Comment:
Sorry this was an old change that I removed in the py-slang branch but I forgot to ask you to update your branch. This breaks py-slang, could you review and approve https://github.com/source-academy/js-slang/pull/1405 which fixes it please?

Reply:
Will do.